demo/demo-springmvc/springmvc-client/src/main/java/org/apache/servicecomb/demo/springmvc/client/TestDateTimeSchema.java

```diff
@@ -92,4 +119,39 @@ private void testDateTimeSchema() {
         dateTimeSchemaInf.postLocalDateTime(localDateTime)
             .format(DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS")));
   }
+
+  private void testDateTimeSchemaMulticast() {
+    DiscoveryContext context = new DiscoveryContext();
+    VersionedCache serversVersionedCache = discoveryTree.discovery(context, "springmvctest", "springmvc", "0+");
+    List<String> enpoints = serversVersionedCache.data();
+
+    for (String endpoint : enpoints) {
+      InvocationContext invocationContext = new InvocationContext();
+      invocationContext.addLocalContext(LoadbalanceHandler.SERVICECOMB_SERVER_ENDPOINT, endpoint);
+      Date date = new Date();
+      TestMgr.check(date.getTime(), dateTimeSchemaWithContextInf.getDate(invocationContext, date).getTime());
```

Comment:
not so good.  
this makes parse endpoint for each invocation?

Reply:
we have to now, because LoadBalanceHandler read string endpoint.

Reply:
when optimize this, LB should be change too.

Reply:
I create a issue for this : https://issues.apache.org/jira/browse/SCB-1831